<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRadioPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_station">
        <property name="text">
         <string>Добавить станцию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="change_station">
        <property name="text">
         <string>Сменить станцию</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settings_btn">
        <property name="text">
         <string>Настройки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="volume_not_adj">
        <property name="text">
         <string>Громкость:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volume_slider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="sync_btn">
        <property name="text">
         <string>Синхронизация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="record_btn">
        <property name="text">
         <string>Запись</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switch_left_btn">
        <property name="text">
         <string>|◄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="play_btn">
        <property name="text">
         <string>►</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switch_right_btn">
        <property name="text">
         <string>►|</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRadio">
    <property name="title">
     <string>Station name:</string>
    </property>
   </widget>
   <addaction name="menuRadio"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
